app/main/presenters/search_summary.py

```diff
@@ -31,7 +24,16 @@ def write_list_as_sentence(input_list, final_conjunction):
             formatted_conjunction = " {} ".format(final_conjunction)
             return formatted_conjunction.join([u', '.join(start), end])
 
-    def __init__(self, results_total, request_args, filter_groups, lots_by_slug):
+    def __init__(self, results_total, request_args, filter_groups, lots_by_slug, markup_tags=True):
+        self.markup_tags = markup_tags
```

Comment:
In my head this name isn't obviously boolean, can we have `include_markup` instead maybe?